NitroxServer/ConsoleCommands/ConfigCommand.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System.Diagnostics;
+using System.IO;
+using System.Reflection;
+using System.Threading;
+using System.Threading.Tasks;
+using NitroxModel.Core;
+using NitroxModel.DataStructures.GameLogic;
+using NitroxModel.Logger;
+using NitroxModel.Serialization;
+using NitroxServer.ConsoleCommands.Abstract;
+using NitroxServer.Serialization;
+
+namespace NitroxServer.ConsoleCommands
+{
+    internal class ConfigCommand : Command
+    {
+        private readonly SemaphoreSlim configOpenLock = new SemaphoreSlim(1);
+
+        public ConfigCommand() : base("config", Perms.CONSOLE, "Opens the server configuration file")
+        {
+        }
+
+        protected override void Execute(CallArgs args)
+        {
+            if (!configOpenLock.Wait(0))
+            {
+                Log.Warn("Waiting on previous config command to close the configuration file.");
+                return;
+            }
+
+            ServerConfig currentActiveConfig = NitroxServiceLocator.LocateService<ServerConfig>();
+            string configFile = Path.Combine(Path.GetDirectoryName(Assembly.GetEntryAssembly()?.Location) ?? "", currentActiveConfig.FileName);
+            if (!File.Exists(configFile))
+            {
+                Log.Error($"Could not find config file at: {configFile}");
+                return;
+            }
+
+            Task.Run(async () =>
+            {
+                try
+                {
+                    await StartProcessAsync("notepad", configFile);
+                }
+                finally
+                {
+                    configOpenLock.Release();
+                }
+                ServerConfig newConfigFromFile = PropertiesWriter.Deserialize<ServerConfig>();
+                if (!ServerConfig.ServerConfigComparer.Equals(currentActiveConfig, newConfigFromFile))
```

Comment:
What about comparing the "last time edited"-time? Fells lighter then the `ServerConfigComparer` class.

Reply:
Definitely lighter but I want to only do this if the values are different from before. Not if it has extra whitespace or saved twice which will change file edit time. I think this is the best way to do that.

Reply:
Is this equality comparator only for showing the log message?  Maybe we should simply always show the log message, i.e.: "if you made changes, restart the server for them to take effect"

Reply:
Changed the message and removed the comparer class.